### ISSUE TITLE
Do the training data for the pretrained OFA include samples from COCO val set?

### ISSUE THREAD

**Issue by ParadoxZW** [OFA-Sys/OFA#348]
Does the pretrain dataset include COCO detection val, VQA val or COCO captioning val (or part of val)? If yes, it means the pretrained model has seen the images from COCO val (even before finetuning) and there is a data leak problem if I want to evaluate pretrained OFA on the validation set of VQA or COCO Captioning?

**Comment by JustinLin610** [comment]
No, such data are not included

**Comment by ParadoxZW** [comment]
Thanks for your reply.
